Content.Server/Atmos/EntitySystems/AtmosPlaqueSystem.cs

```diff
@@ -0,0 +1,92 @@
+using Content.Server.Atmos.Components;
+using Content.Shared.Atmos.Visuals;
+using Robust.Shared.GameObjects;
+using Robust.Shared.IoC;
+using Robust.Shared.Random;
+
+namespace Content.Server.Atmos.EntitySystems;
+
+public sealed class AtmosPlaqueSystem : EntitySystem
+{
+    [Dependency] private readonly IRobustRandom _random = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<AtmosPlaqueComponent, ComponentStartup>(OnPlaqueStartup);
+        SubscribeLocalEvent<AtmosPlaqueComponent, MapInitEvent>(OnPlaqueMapInit);
+    }
+
+    private void OnPlaqueStartup(EntityUid uid, AtmosPlaqueComponent component, ComponentStartup args)
+    {
+        UpdateSign(component);
+    }
+
+    private void OnPlaqueMapInit(EntityUid uid, AtmosPlaqueComponent component, MapInitEvent args)
+    {
+        var rand = _random.Next(100);
+        // Let's not pat ourselves on the back too hard.
+        // 1% chance of zumos
+        if (rand == 0) component.TypeVV = PlaqueType.Zumos;
+        // 9% FEA
+        else if (rand <= 10) component.TypeVV = PlaqueType.Fea;
+        // 45% ZAS
+        else if (rand <= 55) component.TypeVV = PlaqueType.Zas;
+        // 45% LINDA
+        else component.TypeVV = PlaqueType.Linda;
```

Comment:
Maybe set component `component.Type`, not `component.TypeVV`? Tbh this entire VV thing looks really dirty.

Reply:
Last maintainers talked about it AFAIK we didn't like it but VV doesn't really support ECS so we can't do much else for now until it's done.